scripts-dev/lint.sh

```diff
@@ -0,0 +1,3 @@
+isort -y -rc synapse tests scripts-dev scripts
```

Comment:
probably also needs a `set -e` so that it will exit non-zero if `isort` or `flake8` fail. (Alternatively, some more complicated hackery to track if any of the linters fail)